### ISSUE TITLE
Merge - Calculation Mode error

### ISSUE THREAD

**Issue by DanielWZH** [hako-mikan/sd-webui-supermerger#98]
![image](https://github.com/hako-mikan/sd-webui-supermerger/assets/94508650/f9801b71-259e-49b5-89e1-26f5075d6d8a)
The "Normal" Calculation Model performed well while the "CosineA" and "CosineB" experienced error.
It shows tensor cannot access storage directly with .numpy() but should with .detach().numpy()
I have tried to modify the line 237 in merger.py but it cannot work.  :)

**Comment by hako-mikan** [comment]
Tell me your environment, version of torch may be important.
You can find it footer of we-bui.
